src/components/ArticleCard.jsx

```diff
@@ -1,8 +1,31 @@
+import {Link} from 'react-router-dom'
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import Stack from '@mui/material/Stack';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
 
-const ArticleCard = () => {
+const ArticleCard = (props) => {
+    const {title,article_img_url,topic,article_id, author,created_at,votes,comment_count} = props
     return (
-        <section className="ArticleCard">
-
-        </section>
+        
+        <article className="ArticleCard">
+           
+            <Card elevation = {2} sx={{maxWidth: 500}} direction="row">
+            <CardHeader title={title} subheader={topic}/>
+            <CardMedia component="img" height="350" image={article_img_url} alt={`picture of ${title}`}/>
+            <b>Article id: <Link to={`/articles/${article_id}`}>{article_id}</Link></b>
```

Comment:
I had to spend a little bit of time searching for what to click on to go to the article, I would recommend having the `Link` either around the title for each article or if you want it can be wrapped around the whole card so a user can click anywhere on it 💡 
The _article_id_ is less for users to see and more of a backdoor for you as a developer to reference a particular article